sky/packages/sky/lib/src/rendering/proxy_box.dart

```diff
@@ -396,8 +453,9 @@ class RenderClipRRect extends RenderProxyBox {
     assert(_yRadius != null);
   }
 
-  double _xRadius;
+  /// The radius of the rounded corners in the horizontal direction
```

Comment:
I mean, everything is in logical pixels.  It seems like we should say that somewhere more generally.